laravel-app/app/Http/Controllers/CommentController.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Http\Requests\CommentRequest;
+use App\Models\UserPost;
+use App\Models\PostComment;
+use App\Models\User;
+use Illuminate\Contracts\View\View;
+use Illuminate\Http\RedirectResponse;
+
+class CommentController extends Controller
+{
+    public function create(UserPost $post): View
+    {
+        return view('comments.create', compact('post'));
```

Comment:
Hmmm actually now that i noticed i dont prefer to use compact

Reply:
oh is there a better way sir to pass data from controller to view?